messages/agent.generate.spec.md

```diff
@@ -1,41 +1,49 @@
 # summary
 
-Create an Agent spec.
+Generate an agent spec, which is the list of jobs that the agent performs.
 
 # description
 
-Create an Agent spec, which is a list of job titles and descriptions that the agent performs.
+When using Salesforce CLI to create an agent in your org, the first step is to generate the local JSON-formatted agent spec file with this command.
```

Comment:
This is true for the existing commands but not necessarily true with v2.  The v2 APIs allow agent creation from the fields used to generate a spec and not only from a spec.

Reply:
Ditto